src/zoid/buttons/component.jsx

```diff
@@ -1266,13 +1275,28 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
 
       supportedNativeBrowser: {
         type: "boolean",
-        value: isSupportedNativeBrowser,
+        value: ({ props }) => {
+          if (props.fundingSource === FUNDING.VENMO) {
```

Comment:
will we ever hit this condition as the props are passed down to buttons iframe during render 🤔 ?